arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
There is already a swagger spec about the webservices in the same path https://github.com/Azure/azure-rest-api-specs/blob/master/arm-machinelearning/2016-05-01-preview/swagger/webservices.json. If you wish that the generated client should contain the operations of the webservices and commitmentPlans swagger spec then you need to create a composite client. RPs like compute are doing it right now. For more information, please take a look over [here](https://github.com/Azure/azure-rest-api-specs/tree/master/arm-compute#composite-swagger).